downstream/modules/aap-on-azure/proc-azure-locate-aap-marketplace.adoc

```diff
@@ -0,0 +1,20 @@
+////
+Base the file name and the ID on the module title. For example:
+* file name: con-my-concept-module-a.adoc
+* ID: [id="con-my-concept-module-a_{context}"]
+* Title: = My concept module A
+////
+
+[id="proc-azure-locate-aap-marketplace"]
+
+= Locating {PlatformNameShort} in Azure Marketplace
+
+// [role="_abstract"]
+
+
+. In a browser, navigate to the Azure Marketplace.
+. Click *Private Products* from the menu on the left of the screen.
+. Search for {PlatformName}.
+. Click the card that is returned in the search. Be sure to select the official offering from Red Hat.
+. Click *Get it Now*, then *Continue*, then *Create* to initiate the deployment process.
```

Comment:
Suggestion: `Click *Get it Now*, then *Continue*, then *Create* to initiate the deployment process.` > `Click *Get it Now*, *Continue*, and then *Create* to initiate the deployment process.`